lib/components/home/artwork_rail.js

```diff
@@ -54,12 +56,19 @@ class ArtworkRail extends React.Component {
     // pop new view controller
   }
 
+  onGridLayout(event) {
+    console.log('height: ' + event.nativeEvent.layout.height);
+    this.setState({ gridHeight: event.nativeEvent.layout.height });
+  }
+
   renderModuleResults() {
     if (this.props.rail.results && this.props.rail.results.length) {
       return (
-        <View style={[styles.container, { height: this.state.expanded ? 2350 : 500 }]}>
-          <View style={[styles.gridContainer, { height: this.state.expanded ? 2300 : 420 }]}>
-            <Grid artworks={this.props.rail.results}/>
+        <View style={[styles.container, { height: this.state.expanded ? this.state.gridHeight + 150 : 500 }]}>
+          <View style={[styles.gridContainer, { height: this.state.expanded ? this.state.gridHeight + 100 : 420 }]}>
+            <View onLayout={this.onGridLayout.bind(this)}>
```

Comment:
I wonder if this can be replaced with the ES7 syntax `this.onGridLayout.bind(this)` 

can you try this?

```
<View onLayout={onGridLayout}>
```
and above
```
 onGridLayout = (event) => {
      this.setState({ gridHeight: event.nativeEvent.layout.height });
  }
 
```